### PR TITLE
Create privacy policy page embedded into the extension

### PR DESCRIPTION
**Pull request by MikkCZ**
Build an privacy policy page to be shipped with the extension. Part of #146.

Snímek z 2021-05-12 10-23-40

### CODE REVIEW

**Comment by flodolo** [comment]
Leaving the review to Matjaz, given the amount of code. But the page looks good to me, based on the screenshot.

File: src/packages/options/index.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
       <ul>
         <li><a class="open_tour">Tour</a></li>
         <li><a class="open_wiki">Wiki</a></li>
-        <li><a class="privacy-policy" href="https://github.com/MikkCZ/pontoon-addon/blob/master/PRIVACY.md">Privacy</a></li>
+        <li><a class="privacy-policy" href="../../packages/privacy-policy/dist/index.html">Privacy</a></li>
```
Comment (mathjazz) [inline src/packages/options/index.html:22]: Seems like `../privacy-policy/dist/index.html` would also work here?

**Review by mathjazz — COMMENTED** [review]
A high-level newbie question before I take the add-on for a spin - why do you organize the code into a series of packages, which then all need a separate set of config files (for eslint, jest, ts, webpack...), which is pretty WET?

**Comment by MikkCZ** [comment]
> A high-level newbie question before I take the add-on for a spin - why do you organize the code into a series of packages, which then all need a separate set of config files (for eslint, jest, ts, webpack...), which is pretty WET?

That's not a newbie question, but the answer is me being a newbie JS developer once. First the extension was all plain JavaScript, just zip it and run it, no tools needed. Then I wanted to organize the code "by feature" and start using React in some of the places, but I didn't understand the tools and thought that there could only be a single React app built from a single package. So I split it into separate projects using Lerna to orchestrate it.

Now I know and after the last two packages are rewritten to TypeScript, I want to partially undo that historical decision in #135.

File: src/packages/options/index.html (hunk 1 above)
Comment (MikkCZ) [reply src/packages/options/index.html]: Done, and I addressed your comment from #148 as well.

File: PRIVACY.md [hunk 2]
```diff
@@ -1,11 +1,14 @@
 # Privacy Policy
 
-**Pontoon Add-on**:
+**Pontoon Add-on** does not:
 
-- **Does not** collect any personally identifiable data.
-- **Does not** communicate with its author.
-- **Does** download data from external servers, which are necessary to provide its functionality as described below.
-- **Does** store the download data locally as described below.
+- collect any personally identifiable data.
```
Comment (flodolo) [inline PRIVACY.md:5]: Nit: start with uppercase each list item

**Comment by mathjazz** [comment]
Thanks for the update and the explanation. Happy to hear about #135!

I only noticed two minor annoyances:
- The Privacy link on the Settings page opens in the same window, unlike the Tour and Wiki links.
- The Privacy link on the Settings page should have the same padding (or padding-left) as the Tour and Wiki links.

**Review by flodolo — APPROVED** [review]

**Comment by MikkCZ** [comment]
> Thanks for the update and the explanation. Happy to hear about #135!
> 
> I only noticed two minor annoyances:
> 
>     * The Privacy link on the Settings page opens in the same window, unlike the Tour and Wiki links.
> 
>     * The Privacy link on the Settings page should have the same padding (or padding-left) as the Tour and Wiki links.

Done and done. :+1:

**Review by mathjazz — COMMENTED** [review]
I don't understand every single line of the patch, but in general it makes sense and the addon works well. Well done!

**Review by mathjazz — APPROVED** [review]

**Comment by MikkCZ** [comment]
Trust me, I'm an engineer! ;)

**Pull request merged**